service/controller/app/resource/configmap/desired.go

```diff
@@ -40,7 +41,18 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 
 	mergedData, err := r.values.MergeConfigMapData(ctx, cr, cc.AppCatalog)
 	if values.IsNotFound(err) {
-		r.logger.LogCtx(ctx, "level", "warning", "message", fmt.Sprintf("dependent configMaps are not found"), "stack", fmt.Sprintf("%#v", err))
+		r.logger.LogCtx(ctx, "level", "warning", "message", "dependent configMaps are not found")
```

Comment:
The app-operator should cancel the resource when there are no matching user configMaps/secrets.